docs/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" href="//pagecdn.io/lib/swagger-ui/v3.31.1/swagger-ui.css">
+    <title>Flotilla</title>
+
+<body>
+    <div id="openapi">
+        <script src="//pagecdn.io/lib/swagger-ui/v3.31.1/swagger-ui-bundle.js"></script>
+        <script>
+            window.onload = function () {
+                const ui = SwaggerUIBundle({
+                    url: "openapi.yaml",
+                    dom_id: "#openapi"
+                })
+            }
+        </script>
+</body>
```

Comment:
Consider closing the tags. 

```suggestion
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="//pagecdn.io/lib/swagger-ui/v3.31.1/swagger-ui.css">
    <title>Flotilla</title>
</head>
<body>
    <div id="openapi">
    </div>
    <script src="//pagecdn.io/lib/swagger-ui/v3.31.1/swagger-ui-bundle.js"></script>
     <script>
        window.onload = function () {
            const ui = SwaggerUIBundle({
                url: "openapi.yaml",
                dom_id: "#openapi"
            })
        }
        </script>
</body>
```